protos_shared/vpn.proto

```diff
@@ -56,6 +56,7 @@ message Plan {
 message PaymentProviders {
   string name = 1;
   repeated string logoUrls = 3;
+  map<string, string> data = 4;
```

Comment:
This may not be an issue with the new code, but I remember the last time we added data to the PaymentProviders message, that caused old clients to crash because of an InvalidProtocolBufferException: https://github.com/getlantern/lantern-client/pull/1011 and https://github.com/getlantern/lantern-client/pull/1011 so we just want to make sure to check that's no longer the case by installing an old version and then a new version with these changes

Reply:
@atavism  I agree, Do take the test with the old version vs the new version, and let me know if you are getting crash.

Reply:
I installed 7.9.3 and then a new release build with these changes and I didn't run into any issues!